cfe_internal/enterprise/CFE_knowledge.cf

```diff
@@ -200,6 +200,20 @@ bundle agent cfe_internal_setup_knowledge
                     well take care of it separately",
         perms => mog("0550", "root", $(def.cf_apache_group) );
 
+      "$(cfe_internal_hub_vars.docroot)/ldap/." -> { "ENT-9693" }
+        comment => "The ldap directory and it's subdirectories need to be
+                    executable by cfapache",
```

Comment:
I have seen issues with multi-line comments so I try to avoid introducing more.

```suggestion
        comment => concat( "The ldap directory and it's subdirectories need to be ",
                    "executable by cfapache"),
```